schemas/com.snowplowanalytics.snowplow/recoveries/jsonschema/1-0-0

```diff
@@ -0,0 +1,145 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an array of recovery scenarios",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "recoveries",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "array",
+  "items": {
+    "type": "object",
+    "description": "Recovery scenario sum type",
+    "anyOf": [
+      {
+        "properties": {
+          "name": {
+            "description": "The PassThrough recovery scenario doesn't modify the payload",
+            "enum": [
+              "PassThrough"
+            ]
+          },
+          "error": {
```

Comment:
Should we mention format of `error`s in the description?

Reply:
wdym by format?

Reply:
Sorry I though its going to contain complex logic such as `contains("invalid schema") && contains("com.acme")`, but apparently this is going to be simple matcher.